### PR TITLE
Create wake_butterflies.md

### PR DESCRIPTION
**Pull request by aarshita02**
Added a new haiku in wake_butterflies.md file

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku. The test suite on our haiku will let you know which lines are failing, please check the **details** under "All checks have failed" of this pull request to figure out which lines need to be modified.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by aarshita02** [comment]
I have updated the file, it now included 5-7-5 syllables.

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🦋

**Pull request merged**
